plugins-bundled/internal/marketplace-app/src/pages/PluginDetails.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/css';
+
+import { AppRootProps, GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, TabsBar, TabContent, Tab, Icon } from '@grafana/ui';
+
+import { VersionList } from '../components/VersionList';
+import { InstallControls } from '../components/InstallControls';
+import { PLUGIN_ROOT, GRAFANA_API_ROOT } from '../constants';
+import { usePlugin } from '../hooks/usePlugins';
+import { Page } from 'components/Page';
+import { Loader } from 'components/Loader';
+
+export const PluginDetails = ({ query }: AppRootProps) => {
+  const { slug } = query;
+  const [tabs, setTabs] = useState([
+    { label: 'Overview', active: true },
+    { label: 'Version history', active: false },
+  ]);
+  const { status, local, remote, remoteVersions } = usePlugin({ slug });
+  const styles = useStyles2(getStyles);
+
+  const description = remote?.description;
+  const readme = remote?.readme;
+  const version = local?.info?.version || remote?.version;
+  const links = (local?.info?.links || remote?.json?.info?.links) ?? [];
+  const downloads = remote?.downloads;
+
+  if (status === 'LOADING') {
```

Comment:
Consider introducing a constant?

Reply:
I was considering refactoring `status` to `isLoading: boolean;` so there's no need for the comparison operators everywhere. WDYT @hugohaggmark?

Reply:
Sounds great @jackw